message_ix_models/model/water/cli.py

```diff
@@ -194,8 +194,8 @@ def nexus(context: "Context", regions, rcps, sdgs, rels, macro=False):
 
 @cli.command("cooling")
 @common_params("regions")
-@click.option("--rcps", type=click.Choice(_RCPS))
-@click.option("--rels", type=click.Choice(_REL))
+@click.option("--rcps", default="no_climate", type=click.Choice(_RCPS))
```

Comment:
```suggestion
@click.option("--rcps", default="no_climate", type=click.Choice(_RCPS), show_default=True, help="...")
```
Please consider including `show_default` and `help` (but please fill in a relevant explanation for `help`), which would aid people calling `mix-models cooling --help`. Same below.